include/maliput/geometry_base/road_geometry.h

```diff
@@ -134,6 +132,11 @@ class RoadGeometry : public api::RoadGeometry {
     return raw_pointer;
   }
 
+  template <class StrategyT = BruteForceStrategy>
```

Comment:
I think we decided to provide KDTreeStrategy by default as it is the most performant. Right?

Reply:
That is correct. Done.